testdata/node/pod_stat.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Where is this file used ?

Reply:
in comparison with pod_nostat.json, for debug, not directly in the test